src/main/java/edition/academy/seventh/serivce/BookService.java

```diff
@@ -2,28 +2,38 @@
 
 import edition.academy.seventh.database.model.Book;
 import edition.academy.seventh.repository.BookRepository;
+import java.util.List;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-
-import java.util.List;
-
-
 /**
  * Service that provides API responsible for adding books into database.
  *
  * @author Agnieszka Trzewik
  */
 @Service
-public class BooksService {
-
+public class BookService {
   private BookRepository bookRepository;
 
   @Autowired
-  public BooksService(BookRepository bookRepository) {
+  public BookService(BookRepository bookRepository) {
     this.bookRepository = bookRepository;
   }
 
+  /**
+   * Add books to database.
```

Comment:
Adds given books to database (@link Book) 

gdzie linki do powiązanych klas????